src/content/getting-started/architecture/networkplugin-syncer/ovn-kubernetes/_index.en.md

```diff
@@ -34,6 +34,10 @@ create OVN resources necessary for Submariner, including:
 * A `submariner_join` logical switch that connects the `submariner_router`
   with the `ovn_cluster_router`.
 
+{{% notice info %}}
+Requires OVN NorthBound DB version 6.1.0+
```

Comment:
Can we add that its available since OCP4.11+

Reply:
I used this coz it is more generic. OCP 4.11+ is too specific to OCP. I do want to mention it somewhere, but couldn't figure out good place to add it. But now that you mention it, additional comment about OCP version should be fine here.